library/js/xl/jquery-datetimepicker-2-5-4.js.php

```diff
@@ -92,7 +92,9 @@
             <?php } ?>
         <?php } else { ?>
             <?php if ($datetimepicker_formatInput) { ?>
-                format: '<?php echo DateFormatRead("jquery-datetimepicker"); ?> H:i',
+                format: '<?php echo DateFormatRead("jquery-datetimepicker"); ?> g:i a',
+                formatTime:'g:i a',
+                validateOnBlur: false,
```

Comment:
this at least prevents the pm time from converting to am and the `validateOnBlur()` prevents an hour prior time from being selected if clicking into the date field.